includes/class-assets.php

```diff
@@ -283,13 +290,13 @@ public function enqueue_admin_scripts( $force = false ) {
 
 		wp_enqueue_script( 'wpbdp-frontend-js', WPBDP_ASSETS_URL . 'js/wpbdp.min.js', array( 'jquery' ), WPBDP_VERSION, true );
 
-		wp_enqueue_script( 'wpbdp-admin-js', WPBDP_ASSETS_URL . 'js/admin.min.js', array( 'jquery', 'thickbox', 'jquery-ui-sortable' ), WPBDP_VERSION, true );
+		wp_register_script( 'wpbdp-admin-layout-js', WPBDP_ASSETS_URL . 'js/admin-layout.min.js', array( 'jquery' ), WPBDP_VERSION, true );
```

Comment:
Is there a reason to make this a separate file?

Reply:
The main file was getting too big and its a bit all over the place with different duplicate `jQuery(document).ready(function($){` and `(function($) {` called in the file. As a separate file it is easier to find the changes required. But this can be combined into one with a grunt combine config https://www.npmjs.com/package/grunt-combine-files

Reply:
The Grunt build is kind of rigid...maybe later on we can improve this. I have moved the contents to the admin.js file for now

Reply:
Yep. Looks like we need to add a concat option. But for now, I think it's fine to combine them like you did. Thanks!